src/modules/languages/unison.nix

```diff
@@ -0,0 +1,23 @@
+{ pkgs, config, lib, ... }:
+
+let
+  cfg = config.languages.unison;
+in
+{
+  options.languages.unison = {
+    enable = lib.mkEnableOption "Enable tools for Unison development.";
+
+    package = lib.mkOptionn {
```

Comment:
```suggestion
    package = lib.mkOption {
```

Reply:
Ooops, I have no idea how that happened!